services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -55,10 +55,249 @@
 		<property name="serverUri" value="${fcrepo.serverUri}" />
 	</bean>
 
-	<bean id="pids" class="edu.unc.lib.dl.fcrepo4.PIDs">
-		<property name="repository" ref="repository" />
+	<bean id="pids"
+		class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+		<property name="targetClass" value="edu.unc.lib.dl.fcrepo4.PIDs" />
+		<property name="targetMethod" value="setRepository" />
+		<property name="arguments">
+			<list>
+				<ref bean="repository" />
+			</list>
+		</property>
+	</bean>
+	
+	<!-- Solr settings -->
+	<bean id="collectionsPid" class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
```

Comment:
I'm surprised this isn't causing errors since there's no tripleStoreQueryService?  We're going to need to find another way to work this.  Maybe guess is this is going to go away completely and be replaced by a method call on the Repository object or some replacement

From the time being, I'd like to remove this bean and you can probably pass a null into SolrQueryLayerService with something like:
<property name="collectionsPid" value="null" />